testing/xfpga/test_token_list_c.cpp

```diff
@@ -42,7 +42,27 @@ extern pthread_mutex_t global_lock;
 
 using namespace opae::testing;
 
-TEST(token_list_c, simple_case)
+class token_list_c_p : public ::testing::TestWithParam<std::string> {
```

Comment:
Why are we parameterizing these tests? I thought they faked the sysfs structures (rather than relying on mock/real driver). Do they use driver related structures?

Reply:
They needed to be parameterized because they use token_get_parent(), and because token_get_parent() now uses opendir, which requires the mock redirection to be in place before it will work.  These tests were failing until I added the system_->prepare_sysfs(platform_), since the mock sysfs wasn't set up.

Reply:
👍